### ISSUE TITLE
member.displayColor and member.displayHexColor doesn't works

### ISSUE THREAD

**Issue by RobotoSkunk** [discordjs/discord.js#6249]
**Please describe the problem you are having in as much detail as possible:**

I updated the library from v12 to v13-dev to check the new features and I find a bug with the member colors.
First the console returned me `TypeError: member.roles.cache.array is not a function` so I thought that in this version, the function `member.roles.cache.array` was deprecated, then I replaced it. After that, the console returned me `TypeError: sorted.array is not a function` and when I commented the lines with `member.displayHexColor` and `member.displayColor` it disappears.

The bug is caused by the file `.\src\structures\Role.js` in the line 162 because the function `array()` doesn't exists.
https://github.com/discordjs/discord.js/blob/ee3cdc81dd63483f8735265c0237b81219aa6743/src/structures/Role.js#L162

**Sample code:**
```js
const client = new Discord.Client({
	"intents": [Discord.Intents.FLAGS.GUILDS, Discord.Intents.FLAGS.GUILD_MESSAGES]
});
client.once('ready', () => {
	console.log('Bot ready and running!');
	client.user.setPresence({
		"activity":{
			"type":"STREAMING",
			"url":"https://www.youtube.com/watch?v=MobkO51msMI",
			"name":`d!help | robotoskunk.com`
		}
	});
});

client.on('messageCreate', async message => {
	try {
		if (message.content == "color-bug-test") {
			return await message.channel.send(message.member.displayHexColor);
			return await message.channel.send(message.member.displayColor);
		}
	} catch(err) {
		console.log(err);
	}
});

client.login('token goes here').catch((err) => {
	console.log(err);
	process.exit(-1);
});
```

**Further details:**

- discord.js version: 13.0.0-dev.t1627689836.ee3cdc8
- Node.js version: 14.17.4
- Operating system: Windows 10
- Priority this issue should have: as the `array()` function was removed from the Role class, it may be that other functions have been affected.

**Relevant client options:**

- partials: none
- gateway intents: GUILDS, GUILD_MESSAGES
- other: none

**Comment by Jiralite** [comment]
Will be resolved with #6245!

**Comment by RobotoSkunk** [comment]
> Will be resolved with #6245!

Woah, that commit fixed my issue!! Thank you!!!
